lib/spoom/config.rb

```diff
@@ -6,5 +6,6 @@ module Config
     SORBET_CONFIG = "sorbet/config"
     SORBET_GEM_PATH = Gem::Specification.find_by_name("sorbet-static").full_gem_path
     SORBET_PATH = (Pathname.new(SORBET_GEM_PATH) / "libexec" / "sorbet").to_s
+    SPOOM_PATH = (Pathname.new(__FILE__) / ".." / ".." / "..").to_s
```

Comment:
I'm curious what's going on here. I think you are moving back to the root directory of this repo but i don't know how. 

What does the `/` do?

Reply:
`Pathname#/` is an alias to `Pathname#join` :)